lib/ecto/migration/index.ex

```diff
@@ -0,0 +1,192 @@
+defmodule Ecto.Migration.Index do
```

Comment:
As the whole module have name index, there is no need to repeat in every function 'index'

Using 'deleted?' instead of 'is_index_deleted' ?